src/App.js

```diff
@@ -1,13 +1,21 @@
 import React from "react";
-import MainNav from "./components/main-nav";
-import ContactForm from "./components/contact-form";
+import MainNav from "./components/nav/main-nav";
+import ContactForm from "./components/contact-form/contact-form";
+import Header from "./components/header/header";
+import Welcome from "./components/welome/welcome";
+import styled from "styled-components";
 
+const MainStyle = styled.div`
```

Comment:
lets move this into its own file and then import it

Reply:
Ok. I will